client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -66,6 +68,10 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 		return (
 			<Fragment>
 				<DocumentHead title={ translate( 'Summary ‹ Jetpack Onboarding' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Homepage ‹ Jetpack Onboarding' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-onboarding-pageview-tracker)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).